app/models/physical_storage.rb

```diff
@@ -9,10 +9,6 @@ class PhysicalStorage < ApplicationRecord
   has_one :asset_detail, :as => :resource, :dependent => :destroy, :inverse_of => false
 
   has_many :canisters, :dependent => :destroy, :inverse_of => false
-  has_many :computer_system, :through => :canisters
-  has_many :hardware, :through => :computer_system
-  has_many :guest_devices, :through => :canisters
```

Comment:
Do you need to delete these?  Thought you just had to change to `:computer_systems

Reply:
@agrare Initially I thought that just fix the pluralization would do the job, but talking with the team we noted that it didn't make sense to let these relationships on PhysicalStorage since they're already in Canister. So I updated this PR title and description to fit the changes.